stream-chat-android/src/main/java/com/getstream/sdk/chat/style/ScrollbarStyle.kt

```diff
@@ -0,0 +1,58 @@
+package com.getstream.sdk.chat.style
+
+import android.content.res.TypedArray
+import android.graphics.drawable.Drawable
+import android.widget.TextView
+import androidx.annotation.Px
+import androidx.annotation.StyleableRes
+import io.getstream.chat.android.core.internal.InternalStreamChatApi
+
+@InternalStreamChatApi
+public class ScrollbarStyle {
+    private companion object {
+        private const val UNSET_SCROLLBAR_WIDTH = -1
+    }
+
+    private var inputScrollbarEnabled: Boolean = false
+    private var inputScrollbarFadingEnabled: Boolean = false
+    private var inputScrollbarWidth: Int = UNSET_SCROLLBAR_WIDTH
+    private var inputScrollbarThumb: Drawable? = null
+
+    public fun apply(textView: TextView) {
+        textView.isVerticalScrollBarEnabled = inputScrollbarEnabled
+
+        if (inputScrollbarEnabled) {
+            textView.isScrollbarFadingEnabled = inputScrollbarFadingEnabled
+
+            if (inputScrollbarWidth != UNSET_SCROLLBAR_WIDTH) {
+                textView.scrollBarSize = inputScrollbarWidth
+            }
+
+            inputScrollbarThumb?.let {
+                textView.verticalScrollbarThumbDrawable = it
```

Comment:
This call seems to require API 29

https://developer.android.com/reference/android/view/View#setVerticalScrollbarThumbDrawable(android.graphics.drawable.Drawable)